backend/application/syson-application/src/test/java/org/eclipse/syson/application/export/ImportExportTests.java

```diff
@@ -132,4 +132,58 @@ public void checkUseCaseTest() throws IOException {
 
     }
 
+    /**
+     * Test import/export on test file importTest.sysml.
+     *
+     * @see https://github.com/Systems-Modeling/SysML-v2-Release/blob/master/sysml/src/examples/Simple%20Tests/ImportTest.sysml
+     */
+    @Test
+    public void checkImportTest() throws IOException {
+        /*
+         * The file has been modified because a problem has been detected during the export phase that force us to use
```

Comment:
Could you be more precise on the problem you mention here?
You said it should be investigated, but if someone (you or any other person) try to investigate in a few weeks/months, he will not able to start the investigation without more context.

Reply:
done.